mobile/src/foss/java/org/openhab/habdroid/core/CloudMessagingHelper.kt

```diff
@@ -15,25 +15,23 @@ package org.openhab.habdroid.core
 
 import android.content.Context
 import android.content.Intent
-import androidx.annotation.DrawableRes
-
 import org.openhab.habdroid.R
 import org.openhab.habdroid.core.connection.CloudConnection
 
 @Suppress("UNUSED_PARAMETER")
 object CloudMessagingHelper {
 
-    val pushNotificationIconResId: Int
-        @DrawableRes get() = R.drawable.ic_bell_off_outline_grey_24dp
-
     val isSupported: Boolean
         get() = false
 
     fun onConnectionUpdated(context: Context, connection: CloudConnection?) {}
 
     fun onNotificationSelected(context: Context, intent: Intent) {}
 
-    fun getPushNotificationStatus(context: Context): String {
-        return context.getString(R.string.info_openhab_notification_status_unavailable)
+    fun getPushNotificationStatus(context: Context): Pair<String, Int> {
```

Comment:
Maybe use a `data class` for that return value ... advantage over `Pair` would be meaningful naming of the fields.

Reply:
Done